salt/modules/win_useradd.py

```diff
@@ -57,6 +59,59 @@ def __virtual__():
     return False
 
 
+def _get_date_time_format(dt_string):
+    '''
+    Copied from win_system.py (_get_date_time_format)
```

Comment:
It may make more sense here to create a common utility function in salt.utils instead of copying/pasting code around and maintaining the same logic in multiple places.